spec/lib/openstax/cnx/v1/element_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'vcr_helper'
+
+RSpec.describe OpenStax::Cnx::V1::Element, type: :external, vcr: VCR_OPTS do
+
+  let(:cnx_book_id) { '031da8d3-b525-429c-80cf-6c8ed997733a' }
+
+  let(:desired_elements) {
+    [
+      OpenStax::Cnx::V1::Figure,
+      OpenStax::Cnx::V1::Paragraph
+    ]
+  }
+
+  subject(:elements) {
+    OpenStax::Cnx::V1::Book.new(id: cnx_book_id).
+      root_book_part.
+      pages[1].
```

Comment:
If we add a spec later to test that `pages` works on its own, we should do something careful to make sure we only download the book one time (like a `before_all` and an explicit cassette use) so the cassette doesn't grow and grow.